src/genie/libs/parser/iosxe/tests/test_show_ap.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+from unittest.mock import Mock
+
+from genie.metaparser.util.exceptions import SchemaEmptyParserError
+from genie.libs.parser.iosxe.show_ap import ShowApDot115GhzChannel
+
+
+# ==========================================
+# Unit test for 'show ap dot11 5ghz channel'
+# ==========================================
+class TestShowApDot115GhzChannel(unittest.TestCase):
+    """Unit test for 'show ap dot11 5ghz channel'"""
+
+    maxDiff = None
+    empty_output = {'execute.return_value': ''}
+    golden_parsed_output1 = {
+    "channel_assignment": {
+        "chan_assn_mode": "AUTO",
+        "chan_upd_int": 12,
+        "anchor_time_hour": 7,
+        "channel_noise": "Enable",
+        "channel_interference": "Enable",
+        "channel_load": "Disable",
+        "device_aware": "Disable",
+        "clean_air": "Disabled",
+        "wlc_leader_name": "sj-00a-ewlc1",
+        "wlc_leader_ip": "10.7.5.133",
+        "last_run_seconds": 15995,
+        "dca_level": "MEDIUM",
+        "dca_db": 15,
+        "chan_width": "80 MHz",
```

Comment:
`dca_chan_width` is better? I understand you tried to have short name as much as possible. but better to have keyword and `max` or `min` thing in key name.